website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -37,6 +37,11 @@ cd $HOME/projects/memcached-operator
 # so all API groups will be <group>.example.com
 operator-sdk init --domain example.com --repo github.com/example/memcached-operator
 ```
+`--domain` will be used as the prefix of the API group your custom resources will be created in.
+API groups are a mechanism to group portions of the Kubernetes API. You're probably already familiar with
+some of the core Kubernetes API groups, such as `apps` or `rbac.authorization.k8s.io`. You should name your
+domain to group your resource types in meaningful group(s), both for ease of understanding and because these
+groups determine how access can be controlled to your resource types using RBAC. For more information, see [the core Kubernetes docs](https://kubernetes.io/docs/reference/using-api/#api-groups) and [the Kubebuilder docs](https://book.kubebuilder.io/cronjob-tutorial/gvks.html).
```

Comment:
Just to clarify:

>  You should name your
> domain to group your resource types in meaningful group(s), both for ease of understanding and because these
> groups determine how access can be controlled to your resource types using RBAC.
> [@jberkhahn](https://kubernetes.slack.com/team/U7Q21LH2S)

 I do not agree with the above text added in the PR. It is not a correct info and that is the reason I do not agree with your PR.


```suggestion
> Note: By following the [Operators pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) you will check in this tutorial that you will be able to extend the k8s api and create your own APIs. These APIs will be grouped by via the [ApiGroup](https://kubernetes.io/docs/reference/using-api/#api-groups). Therefore, you will see that the value informed via the `--domain` flag will be the subdomain of your APIs. In this case, you probably want to look for something meaningful to your organization and/or project.
```